### PR TITLE
Update aws NLB blog with correct instructions

### PR DESCRIPTION
**Pull request by sdake**

### CODE REVIEW

**Comment by geeknoid** [comment]
Deploy preview for *preliminary-istio* failed.

Built with commit 01cf7ff5454b81895e23f539a9b66230bddc75da

https://app.netlify.com/sites/preliminary-istio/deploys/5c3f528fe6195200081fb423

File: content/blog/2018/aws-nlb/index.md [hunk 1]
```diff
@@ -81,32 +82,19 @@ You need to apply policy on the master role in order to be able to provision net
 
 1. Your policy is now attach to your master node.
 
-## Rewrite Istio Ingress Service
+## Generate the Istio manifest
 
-You need to rewrite ingress service with the following:
```
Comment (frankbu) [inline content/blog/2018/aws-nlb/index.md]: Probably would be good to add a sentence ahead of this one that explains that you need to annotate the istio-ingress service to use the NLB load balancer.

File: content/blog/2018/aws-nlb/index.md [hunk 2]
```diff
@@ -81,32 +82,19 @@ You need to apply policy on the master role in order to be able to provision net
...
+gateways:
+  istio-ingressgateway:
+    serviceAnnotations:
+      service.beta.kubernetes.io/aws-load-balancer-type: "nlb"
+{{< /text >}}
+
+Generate a manifest with Helm:
+
+{{< text bash >}}
+helm template install/kubernetes/helm/istio --namespace istio -f override.yaml > $HOME/istio.yaml
```
Comment (frankbu) [inline content/blog/2018/aws-nlb/index.md]: missing $ at start of line

**Pull request merged**